### DISCUSSION TITLE
build ios simulator on windows?

### DISCUSSION THREAD

**Discussion by shaymolho** [airsdk/Adobe-Runtime-Support#2909] [Q&A] [answered]
Looking at the [docs ](https://help.adobe.com/en_US/air/build/WS901d38e593cd1bac1e63e3d128cdca935b-8000.html):

> when using ADT with the iOS Simulator, you must always include the -platformsdk option, specifying the path to the iOS Simulator SDK

what would this path be on windows?

**Comment by ajwfrost** [comment, marked as answer]
There isn't an iOS simulator on Windows (afaik?!) so those build types aren't supported...
